src/overview.md

```diff
@@ -32,7 +32,10 @@ we'll talk about that later.
 - The token stream passes through a higher-level lexer located in
   [`rustc_parse`] to prepare for the next stage of the compile process. The
   [`StringReader`] struct is used at this stage to perform a set of validations
-  and turn strings into interned symbols (_interning_ is discussed later).
+  and turn strings into interned symbols (_interning_ is discussed later). 
+  string interning is a method of storing only one copy of each distinct 
+  string value, which must be immutable.
+  [read more](https://en.wikipedia.org/wiki/String_interning)
```

Comment:
```suggestion
  [String interning] is a way of storing only one immutable copy of each distinct string value.
```
and then add a reference link at the bottom:
```
  [String interning]: https://en.wikipedia.org/wiki/String_interning)
```

Reply:
You didn't add the reference at the bottom. It should go near the other links.

Reply:
you mean add `[String interning]: https://en.wikipedia.org/wiki/String_interning)` under `read more`?

Reply:
Please put it with the other links for the paragraph: see lines 101-115

Reply:
You undid the previous changes when you added the link.

Reply:
yeah but now that change is back

Reply:
I mean this change: https://github.com/rust-lang/rustc-dev-guide/pull/960/files#r523762736
It's disappeared.

Reply:
fixed

Reply:
Can you please only show 'string interning' once instead of twice? Right now there are 5 lines related to it, I would rather have 1 line like it was before. Or in other words, my suggestion should replace the first change, not add to it.

Reply:
ok sure